simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in

```diff
@@ -546,8 +546,6 @@ exportedFrames:
   - frameName: SCSYS_L_FOOT_SKIN_22
     frameReferenceLink: l_foot
     exportedFrameName: l_sole_skin_22
-@XSENS_IMU_FRAME@
```

Comment:
@nunoguedelha could you explain the significance of this tag `@XSENS_IMU_FRAME@` ?

Reply:
It is a cmake variable set in the `simmechanics/CMakeLists.tx` for adding the XSENS IMU sensor parameters only in the case of the iCub2.5+ (refer to [this comment](https://github.com/robotology/icub-model-generator/issues/114#issuecomment-519608586)).

Reply:
As discussed F2F, the tag @XSENS_IMU_FRAME@ shall be added back.

Reply:
Sorry, let me correct the misunderstanding:
- in the `sensor:` section, the tag `@XSENS_IMU_SENSOR@` adds the parameters of the accelerometer integrated in the XSENS IMU.
- in the `exportedFrames` section, the tag `@XSENS_IMU_FRAME@` adds the zero-mass frame `root_link_imu_frame` in the URDF, also required by iDynTree.